core/src/main/scala/org/broadinstitute/dsde/rawls/Boot.scala

```diff
@@ -28,11 +27,10 @@ import spray.can.Http
 import spray.http.StatusCodes
 import spray.json._
 
-import scala.concurrent.duration._
-import scala.util.{Try, Failure, Success}
-
-import scala.concurrent.ExecutionContext.Implicits.global
 import scala.collection.JavaConversions._
+import scala.concurrent.ExecutionContext.Implicits.global
```

Comment:
Also IIRC Rawls uses the `blocking` construct so if you ever *do* do what @rtitle suggests note that it only does anyting on the global ec